pontoon/base/models.py

```diff
@@ -2858,15 +2859,24 @@ def save(self, update_stats=True, failed_checks=None, *args, **kwargs):
                     rejected=False,
                 )
                 .exclude(pk=self.pk)
-                .update(
-                    approved=False,
-                    approved_user=None,
-                    approved_date=None,
-                    rejected=True,
-                    rejected_user=self.approved_user,
-                    rejected_date=self.approved_date,
-                    fuzzy=False,
+            )
+
+            # Log that all those translations are rejected.
+            for t in approved_translations:
+                log_action(
+                    'translation:rejected',
+                    self.approved_user or self.user,
+                    translation=t,
```

Comment:
We should use `bulk_create` here.

Reply:
I object, in this specific case, there should always be only one translation there. It's done in a loop because it's easier to do it this way, but there should never be more than one approved translation when approving a new one, so there should never be more than one rejection event to log.

Reply:
Good point, sorry for the confusion!